src/BinSkim.Rules/PERules/WindowsBinaryAndPdbSkimmerBase.cs

```diff
@@ -80,10 +80,6 @@ public sealed override AnalysisApplicability CanAnalyze(BinaryAnalyzerContext co
                 // Could not locate the PDB for '{0'}. Probing details:{1}
                 : RuleResources.PdbLoadFailed;
 
-            FailureLevel failureLevel = pdbLoadSucceeded
-                ? FailureLevel.Note
-                : FailureLevel.Warning;
-
             context.Logger.LogConfigurationNotification(

```

Comment:
If we check, failurelevel is hardcoded as warning. Should we use the variable that I just removed?